src/sonic-host-services/scripts/caclmgrd

```diff
@@ -538,7 +542,8 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
 
                 # Obtain default IP protocol(s) and destination port(s) for this service
                 ip_protocols = self.ACL_SERVICES[acl_service]["ip_protocols"]
-                dst_ports = self.ACL_SERVICES[acl_service]["dst_ports"]
+                if "dst_ports" in self.ACL_SERVICES[acl_service]:
```

Comment:
if ACL_RULE does not have a dst_ports specified, does it mean match on only IP address and accept OR raise it as a failure? Its not clear from the example, hence the question.

Reply:
@prsunny if ACL_RULE does not have dst_ports, it will raise a failure, actually a traceback in syslog.
Should we expect this scenario? No port defines in ACL_RULE for cacl.

```
Sep  1 03:15:12.654833 vlab-03 INFO caclmgrd[3816987]: Translating ACL rules for control plane ACL 'EXTERNAL_CLIENT' (service: 'EXTERNAL_CLIENT')
Sep  1 03:15:12.655089 vlab-03 INFO caclmgrd[3816987]: Exception in thread Thread-2:
Sep  1 03:15:12.655141 vlab-03 INFO caclmgrd[3816987]: Traceback (most recent call last):
Sep  1 03:15:12.655229 vlab-03 INFO caclmgrd[3816987]:   File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
Sep  1 03:15:12.670835 vlab-03 INFO caclmgrd[3816987]:     self.run()
Sep  1 03:15:12.670923 vlab-03 INFO caclmgrd[3816987]:   File "/usr/lib/python3.9/threading.py", line 892, in run
Sep  1 03:15:12.671004 vlab-03 INFO caclmgrd[3816987]:     self._target(*self._args, **self._kwargs)
Sep  1 03:15:12.671048 vlab-03 INFO caclmgrd[3816987]:   File "/usr/local/bin/caclmgrd", line 720, in check_and_update_control_plane_acls
Sep  1 03:15:12.672393 vlab-03 INFO caclmgrd[3816987]:     self.update_control_plane_acls(namespace)
Sep  1 03:15:12.672535 vlab-03 INFO caclmgrd[3816987]:   File "/usr/local/bin/caclmgrd", line 669, in update_control_plane_acls
Sep  1 03:15:12.672602 vlab-03 INFO caclmgrd[3816987]:     iptables_cmds, service_to_source_ip_map  = self.get_acl_rules_and_translate_to_iptables_commands(namespace)
Sep  1 03:15:12.672676 vlab-03 INFO caclmgrd[3816987]:   File "/usr/local/bin/caclmgrd", line 608, in get_acl_rules_and_translate_to_iptables_commands
Sep  1 03:15:12.672718 vlab-03 INFO caclmgrd[3816987]:     for dst_port in dst_ports:
Sep  1 03:15:12.672802 vlab-03 INFO caclmgrd[3816987]: UnboundLocalError: local variable 'dst_ports' referenced before assignment
```